telemetry/sql/000007_error_sending_envelope.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS errorSendingEnvelope (
```

Comment:
why do we need this table? it doesn't seem needed for dogfooding and maybe can be combined with other envelope tables?

Reply:
removed as it will be handled in separate PR after this one